keras_contrib/applications/resnet.py

```diff
@@ -113,27 +147,45 @@ def _shortcut(input_feature, residual, conv_name_base=None, bn_name_base=None):
     shortcut = input_feature
     # 1 X 1 conv if shape is different. Else identity.
     if stride_width > 1 or stride_height > 1 or not equal_channels:
-        print('reshaping via a convolution...')
+        if verbose:
+            print("    SHORTCUT not right: input -", shortcut.shape, "residual -", residual.shape)
```

Comment:
I like this expanded comment. Could it include a suggestion on what a user can do to fix the problem?

Reply:
This is not a user-related problem, there is nothing to be fixed.

If the input_feature and residual have unequal channels, then the skip connection cannot be made. In this case, the input_feature is transformed to have the same number of channels as the residual and they are added to make a skip connection.